src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java

```diff
@@ -104,10 +120,8 @@ public void download(final String filePath) throws IOException {
      */
     public String authGetToken() throws IOException {
         String sURL = "http://localhost:8080/authenticate";

```

Comment:
Could be set as a final attribute or be declared in a properties file.

Reply:
changed it.